File: Task/Rainbow/Python/rainbow.py
from colored import Fore, Style
red: str = f'{Fore.rgb(255, 0, 0)}'
orange: str = f'{Fore.rgb(255, 128, 0)}'
yellow: str = f'{Fore.rgb(255, 255, 0)}'
green: str = f'{Fore.rgb(0, 255, 0)}'
blue: str = f'{Fore.rgb(0, 0, 255)}'
indigo: str = f'{Fore.rgb(75, 0, 130)}'
violet: str = f'{Fore.rgb(128, 0, 255)}'
print(f'{red}R{Style.reset}' + f'{orange}A{Style.reset}' + f'{yellow}I{Style.reset}' + f'{green}N{Style.reset}' + f'{blue}B{Style.reset}' + f'{indigo}O{Style.reset}' + f'{violet}W{Style.reset}')

File: Task/Rainbow/Zen-C/rainbow.c
fn main() {
    let colors: (int, int, int)[7] = [
        (255,   0,   0), // red
        (255, 128,   0), // orange
        (255, 255,   0), // yellow
        (  0, 255,   0), // green
        (  0,   0, 255), // blue
        ( 75,   0, 130), // indigo
        (128,   0, 255)  // violet
    ];
    let s = "RAINBOW";
    for i in 0..7 {
        let fore = "\e[38;2;{colors[i].0};{colors[i].1};{colors[i].2}m";
        print "{fore}{s[i]:c}";
    }
    println "";
}
